library/java/org/chromium/net/impl/NativeCronetEngineBuilderImpl.java

```diff
@@ -61,6 +78,17 @@
    */
   public NativeCronetEngineBuilderImpl(Context context) { super(context); }
 
+  /**
+   * Indicates to skip the TLS certificate verification.
+   *
+   * @return the builder to facilitate chaining.
+   */
+  @VisibleForTesting
+  public CronetEngineBuilderImpl setMockCertVerifierForTesting() {
```

Comment:
I moved this here because this is an EnvoyMobile enum that is being use, and this is the only class bridging EM and Cronet for the Configuration.

The second weaker reason is that CronetEngineBuilderImpl (where that method was before this PR) is also used by the Java implementation of Cronet, where it makes no sense.